.github/workflows/releasedevhelm.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Update the helm charts 0-latest
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+    paths:
+      - helm-charts/**
+env:
+  CHARTS_DIR: "helm-charts"
+
+jobs:
+  run-on-merge:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+
+      - name: Get the modified charts list
+        run: |
+          echo "This job runs after a PR is merged!"
+          if grep -r '^version: ' `find helm-charts/ -name "Chart.yaml"` | grep -v "0-latest"; then
+            echo "Version check failed, only update 0-latest"
+            exit 1
+          fi
```

Comment:
can we add this chart version check to `.github/workflows/pr-chart-validate.yaml` ? so all charts in main branch should have 0-latest version.